website/common/script/content/gear/sets/armoire.js

```diff
@@ -1806,53 +1822,62 @@ const releaseDates = {
   schoolUniform: '2024-01-04T08:00-05:00',
   whiteLoungeWear: '2024-02-06T08:00-05:00',
   hatterSet: '2024-03-05T08:00-05:00',
-  optimistSet: '2024-03-05T00:00-05:00',
+  optimistSet: '2024-04-04T08:00-04:00',
+  pottersSet: '2024-05-07T08:00-04:00',
 };
 
-forEach({
```

Comment:
This one I _will_ request you remove, as leaving it would necessitate a review and retest of a bunch of code greatly out of scope for this PR

Reply:
I believe I have corrected this--I removed the curly bois surrounding the `for each` statement.

Reply:
went to `release` and grabbed the code from there.